crates/bindings-csharp/BSATN.Runtime/BSATN/Int128.cs

```diff
@@ -0,0 +1,128 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if !NET7_0_OR_GREATER
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+
+namespace System
```

Comment:
This would be nice, but might be problematic if another package that user depends on declares its own `System.Int128` polyfill. It's ok to declare `internal` System types for things like marker attributes, but public types can lead to all sorts of conflicts.

I think we should keep this type under our own `SpacetimeDB` namespace.